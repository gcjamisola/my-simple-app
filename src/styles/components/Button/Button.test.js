import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {Button} from './index';

test('it works', () => {
	const tree = renderer.create(<Button/>).toJSON();
	expect(tree).toMatchSnapshot();
});