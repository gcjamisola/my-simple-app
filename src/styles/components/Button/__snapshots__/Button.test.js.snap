// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background: #2471FF;
  border: none;
  padding: 10px 20px 10px 20px;
  border-bottom: 3px solid #5994FF;
  border-radius: 3px;
  color: #D2E2FF;
}

.c0:hover {
  background: #6B9FFF;
  color: #fff;
}

<button
  className="c0"
/>
`;
